### ISSUE TITLE
Onedrive file not syncing 

### ISSUE THREAD

**Issue by JD-aus** [PhilippC/keepass2android#1966]
Hi,

I'm using the onedrive connection to the apps own folder, when I update the file using keepassxc, the app is saved into onedrive app folder and I can see the file timestamp change when viewed online, but when I try to sync in this app, it tells me files are in sync, but the latest entries are not there ... super annoying, as I love this app.

Is there a way of enabling debug logs that I can take a look to see what's happening (or not in this case)

It may very well be me, but I have added the same onedrive file again without a problem, but still not in sync...

**Comment by JD-aus** [comment]
I have finally got the latest version of the file to be available in the app, by using the "OneDrive (My Files)" option, instead of the "OneDrive (Keepass2Android App folder)" option when opening the file ... it appears that option is no longer available for me ... i would still like to troubleshoot this if someone can tell me how where log files are or how to enable them ...

**Comment by PhilippC** [comment]
you can enable debug-logging in settings - app.

**Comment by JD-aus** [comment]
Yeah I should have seen that ... woods... just hit send log, which didn't appear to do anything, so let me know if you need anything more ...

**Comment by PhilippC** [comment]
please try again after the next app update using the (then updated) built-in OneDrive implementation.
